mavros/src/plugins/command.cpp

```diff
@@ -64,9 +64,12 @@ class CommandPlugin : public plugin::PluginBase {
 		PluginBase::initialize(uas_);
 
 		cmd_nh.param("use_comp_id_system_control", use_comp_id_system_control, false);
-		double command_ack_timeout_ms;
-		cmd_nh.param("command_ack_timeout", command_ack_timeout_ms, ACK_TIMEOUT_MS_DEFAULT);
-		command_ack_timeout_dt = ros::Duration(command_ack_timeout_ms / 1000);
+
+		double command_ack_timeout;
```

Comment:
```
double foo;

nh.param("bar", &bar, 1);
nh.param("foo", &foo, 2);

baz = ros::Ducration(foo);
```

Reply:
yes, thanks. Got and fixed it.

Btw, is here any style guides? It's a bit frustrating now, when I should look through another source files in order to understand style.

Reply:
No, we don't have one. I tried to setup uncrustify to follow my style, and it's fine most time, but it can't reorganise code blocks like above.

Reply:
The code style guide will work fine in this kind of situations.
I mean, it's really difficult to format this code automatically, but in the code style you can just describe it with the words and examples, so none can argue it.
Also, you can fill the code style guide step by step and you shouldn't reformat all existing code immediately. Just format the new code according to the guide and reformat the old code, if you are touching it. I can make a PR with a draft of code style guide, if you want.

Btw, thanks for merge.